src/main/java/io/openliberty/tools/common/plugins/util/DevUtil.java

```diff
@@ -1354,13 +1354,17 @@ private void startContainer() {
     private Process getRunProcess(String command) throws IOException {
         ProcessBuilder processBuilder = new ProcessBuilder();
         processBuilder.command(getCommandTokens(command));
+        Map<String, String> env = processBuilder.environment();
         if (!OSUtil.isLinux()){
-            Map<String, String> env = processBuilder.environment();
             if (!env.keySet().contains("DOCKER_BUILDKIT")) { // don't touch if already set
                 env.put("DOCKER_BUILDKIT", "0"); // must set 0 on Windows VMs
                 debug("Generating environment for docker build & run: DOCKER_BUILDKIT=0");
             }
         }
+        if (!env.keySet().contains("DOCKER_SCAN_SUGGEST")) { // don't touch if already set
+            env.put("DOCKER_SCAN_SUGGEST", "false"); // Disable stderr contents from showing as error message
+            debug("Generating environment for docker build & run: DOCKER_SCAN_SUGGEST=false");
```

Comment:
If we are on a non-Linux system, are we going to get two "Generating environment for docker build & run:" messages? The one at line 1361 and the one here?

Reply:
I guess it's just debug messages. So that should be fine.